### PR TITLE
Add security section, fixed respec issue, changed affiliation

### PR DESCRIPTION
**Pull request by yoavweiss**
This PR addresses mostly the lack of security/privacy considerations section, as pointed out in the [TAG review](https://github.com/w3ctag/design-reviews/issues/241#issuecomment-434604343)

It addresses Fingerprintability as well as abusive use by ad networks. On top of that, while I was there, I also fixed a couple of admin stuff such as affiliation and things respec was complaining about.

### CODE REVIEW

**Comment by yoavweiss** [comment]
@lknik - can you take a look?

File: index.html [hunk 1]
```diff
@@ -506,6 +507,34 @@ <h2>Safe usage</h2>
...
+    This is not dissimilar to other web platform features, but with one important distinction: 
+    this information is exposed at the HTTP layer, allowing the server implementation to discover discrepancies between the User-Agent string the request is sent with and the browser that sent the request. 
+    It is not clear that this is a significant issue, but it is something for implementing user agents (and user agents which try to spoof others) to be aware of.
+    </p>
+    <h3>Negative ecosystem side-effects</h3>
+    <p>
+    There's a risk that monetizing third-party components, and ad networks in particular, will require publishers to use the feature in order to upgrade the priority of their resource requests.
+    While they certainly can do that, and enforce such a requirement by inspecting the HTTP/2 priorities of the resource requests they receive, the risk for that is rather small.
+    <ul>
+        <li><p>Such "priority upgrade" is already available in the platform today, e.g. when switching scripts from blocking to deferred or async ones.
```
Comment (domfarolino) [inline index.html]: nit: e.g. => e.g.,

**Review by domfarolino — APPROVED** [review]
LGTM, thanks for doing this! Will wait for Lukasz's feedback.

File: index.html (hunk 1 above)
Comment (yoavweiss) [reply index.html]: ack

**Review by addyosmani — APPROVED** [review]
Changes LGTM. Thank you for working on this addition, Yoav! Will circle back if there's further feedback from Lukasz.

**Comment by lknik** [comment]
I'll have a look (though no ETA known today).

**Comment by lknik** [comment]
Hi, looks more or less good. However: 

- if you think HTTPS/Tor would ship this (i.e. assumption of encrypted traffic all the way, similarly even for HTTPS, perhaps! But Tor is the "strongest" threat model), it's both a server and possibly a passive observer too that can potentially look at the patterns

File: index.html [hunk 2]
```diff
@@ -506,6 +507,34 @@ <h2>Safe usage</h2>
...
   </section>
 
+  <section>
+    <h2>Security and Privacy Considerations</h2>
+    <h3>Fingerprintability</h3>
+    <p>This specification does not expose any new information about the user and their preferences, so does not increase the risk of user fingerprinting.
+    At the same time, it does expose priority information to web servers that enables them to distinguish between browsers that support the feature and ones that do not support it.
+    This is not dissimilar to other web platform features, but with one important distinction: 
+    this information is exposed at the HTTP layer, allowing the server implementation to discover discrepancies between the User-Agent string the request is sent with and the browser that sent the request. 
+    It is not clear that this is a significant issue, but it is something for implementing user agents (and user agents which try to spoof others) to be aware of.
```
Comment (yoavweiss) [inline index.html:517]: @lknik - Added a sentence calling out network observers specifically, let me know if it covers your concerns

Comment (lknik) [reply index.html:517]: you mean like, passive network observers?

Comment (yoavweiss) [reply index.html:517]: yeah (the sentence below)

**Comment by yoavweiss** [comment]
Merged as this seems like an improvement on previous state. @lknik - let me know if you have any further concerns and I'd be happy to address them in a followup

File: index.html (hunk 2 above)
Comment (lknik) [reply index.html:517]: I'm not sure if webservers cannot observe if the browser supports the feature or not (different priorities used), but it is not the main concern, I guess. Not sure we need the discussion on UA - up to you. Basically it's whether it makes sense to document that passive attackers maybe can spot differences (https/tor-like scenarios). I'm not particularly concerned with fingerprinting beyond that.

Comment (domfarolino) [reply index.html:517]: > I'm not sure if webservers cannot observe if the browser supports the feature or not

I believe you're saying you think webservers _may_ be able to determine whether a UA supports the feature. I think that is currently what Yoav mentioned in the spec, no? Either way, any change in priority doesn't necessarily mean a UA supports this particular feature. Chrome, for example, runs all kinds of experiments that may touch the priorities of certain resources, for a certain percentage of users etc.

Comment (lknik) [reply index.html:517]: But as I said it's not the main concern, (1/0 supported or not).

**Pull request merged**